src/test/java/com/github/dockerjava/client/AbstractDockerClientTest.java

```diff
@@ -60,7 +60,7 @@ private DockerClientConfig config() {
 
     protected DockerClientConfig config(String password) {
         DockerClientConfig.DockerClientConfigBuilder builder = DockerClientConfig.createDefaultConfigBuilder()
-                .withServerAddress("https://index.docker.io/v1/");
+                .withServerAddress("https://index.docker.io/v1/").withMaxTotalConnections(5).withMaxPerRouteConnections(5);
```

Comment:
Hi Markus,

My apologies, I’m still quite new to github.
I believe it should be correct now.

Best regards,

Eric

On 7 Apr 2015 at 14:40:24, Eric Fjøsne (eric.fjosne@gmail.com) wrote:

Well spotted !

Done.

Thanks !
On 7 Apr 2015 at 14:23:49, marcuslinke (notifications@github.com) wrote:

In src/test/java/com/github/dockerjava/client/AbstractDockerClientTest.java:

> @@ -60,7 +60,7 @@ private DockerClientConfig config() {
>    
>      protected DockerClientConfig config(String password) {
>          DockerClientConfig.DockerClientConfigBuilder builder = DockerClientConfig.createDefaultConfigBuilder()
> -                .withServerAddress("https://index.docker.io/v1/");
> +                .withServerAddress("https://index.docker.io/v1/").withMaxTotalConnections(5).withMaxPerRouteConnections(5);

This should not be needed. Could you remove this please?

—
Reply to this email directly or view it on GitHub.